recipe/0001-Update-sklearn-api.patch

```diff
@@ -1,25 +0,0 @@
-From 5b6401a88d89a7c0cded7a83fac0cec379d1d249 Mon Sep 17 00:00:00 2001
-From: John Lees <lees.john6@gmail.com>
-Date: Wed, 23 Aug 2023 16:55:04 +0100
-Subject: [PATCH] Update sklearn api
```

Comment:
Appears this patch was never applied since it was added in PR: https://github.com/conda-forge/hdbscan-feedstock/pull/61

Maybe this should go upstream?

In any event, have dropped it as it is unused

Reply:
Yes @lmcinnes fixed this upstream so correct to drop